generator/lib/behavior/i18n/I18nBehaviorObjectBuilderModifier.php

```diff
@@ -183,7 +183,7 @@ protected function addTranslatedColumnSetter(Column $column)
             $objectBuilder->addMutatorOpenOpen($functionStatement, $column);
         }
         $comment = preg_replace('/^\t/m', '', $comment);
-        $comment = str_replace('@return     ' . $i18nTablePhpName, '@return     ' . $tablePhpName, $comment);
+        $comment = str_replace('@return ' . $i18nTablePhpName, '@return ' . $tablePhpName, $comment);
         $functionStatement = preg_replace('/^\t/m', '', $functionStatement);
```

Comment:
Looks like the original comment comes from PHP5ObjectBuilder::addMutatorComment().
No need for a regex indeed, just the str_replace with this little modification.